ls1intum/Artemis
# Simplify prometheus config.

### Checklist

### Motivation and Context
This is a bugfix PR of #1845: The introduced `PrometheusAuthConfiguration` seems to be unreliable, therefore now the default `SecurityConfiguration` is used.

### Description
Move the ip-address based authentication to the existing `SecurityConfiguration`.

### Steps for Testing (locally)
**Note**: The testing steps are identical to these of #1845
1. Change the config value `management.metrics.export.prometheus.enabled` to `true`.
2. If necessary, adjust the value of `spring.prometheus.monitoringIp` to match the IP address used to access Artemis.
3. Navigate to `ARTEMIS_IP:8080/management/prometheus` (where `ARTEMIS_IP` is `127.0.0.1` when using the same device) and verify that something like this is shown:
![image](https://user-images.githubusercontent.com/5084100/87094389-1a686980-c23f-11ea-95ee-9b7cd8e16a04.png)

4. Change to config value of `spring.prometheus.monitoringIp` to a different ip address & reboot Artemis.
5. Verify that the metrics can not be accessed anymore:
![image](https://user-images.githubusercontent.com/5084100/87251390-9f37cb00-c46b-11ea-98f4-adf0abc664a1.png)
